bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/discovery/SmartHomeDevicesDiscovery.java

```diff
@@ -92,8 +86,7 @@ public void deactivate() {
     protected void startScan() {
         stopScanJob();
         removeOlderResults(activateTimeStamp);
```

Comment:
I changed the activateTimeStamp to a static variable, otherwise the Discover does not work if multiple accounts are created